frontend/index.html

```diff
@@ -468,8 +529,91 @@ <h2>Contact</h2>
         const study_name = first_domain.substr(0,openpath_index);
         return study_name;
       }
+
+      var curr_lang = "en"; // Default language is set to English
+
+      function getLang(){
+
+      var selectBox = document.getElementById("buttonBox");
+      var selectedValue = selectBox.options[selectBox.selectedIndex].value;
+      curr_lang = selectedValue;
+      localStorage.setItem("langValue", curr_lang);
```

Comment:
you are setting the `langValue` into `localStorage` here, but where do you get it? I don't see any other references to `localStorage` in this codebase.

Reply:
Yes. This local variable is just to let us know that the button is working properly whenever the user changes language. This will be deleted.